### PR TITLE
Ui tweaks

### PR DESCRIPTION
**Pull request by feiss**
polished css and added little tweaks

### CODE REVIEW

**Comment by feiss** [comment]
Screenshot 2019-11-14 at 10 09 27

**Comment by takahirox** [comment]
Thanks, looking very nice!

File: panel.js [hunk 1]
```diff
@@ -624,7 +606,7 @@ ConfigurationManager.createFromJsonFile('./devices.json').then(manager => {
   // set up devices select element
 
   const devices = manager.devices;
-  const deviceKeys = Object.keys(devices).sort();
+  const deviceKeys = Object.keys(devices);
```
Comment (takahirox) [inline panel.js]: Any special reasons why you've removed `.sort()`?

Comment (feiss) [reply panel.js]: So you can have more control on the order of items. For example, "None" should be the first in the list, not lost in the middle. There's almost no benefit of an alphabetic ordering vs  manual and sort things so they can be found easier (Oculus devices together, or 3dof vs 6dof together..). Also, with manual ordering you can layout them alphabetically if you want! :)

Comment (takahirox) [reply panel.js]: Is it guaranteed in JavaScript spec that the order returned from `Object.key()` is consistent?

Comment (feiss) [reply panel.js]: Haha lol, you are right, it is not! I guess I saw an array there, sorry. How dumb..

Comment (takahirox) [reply panel.js]: No worry, I'll fix in my side if needed. And putting None at top of the list is what I've been thinking, too!

**Pull request merged**